src/PrestaShopBundle/Form/Admin/Sell/Discount/CreateDiscountType.php

```diff
@@ -83,7 +83,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'choice_name' => 'type',
                 'expanded' => true,
                 'multiple' => false,
-                'data' => $transformedData['cart_discount'],
+                'data' => $transformedData['cart_level'],
```

Comment:
```suggestion
                'data' => $transformedData[DiscountType::CART_LEVEL],
```
We should rely on the const here, this way if we search for the usage of this value (or want to remove it) we will find them more easily